### ISSUE TITLE
Disconnect from Sync screen does not receive keyboard focus when opened

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa#636]
STR

1. Create an account, attach to Sync.
2. Open /settings
3. Open Devices & apps
4. Click on the "Disconnect" button next to a device.
5. Try to tab around using the keyboard. The page underneath scrolls.

This makes it difficult for screen reader users to use the functionality.

![disconnect-device-focus](https://user-images.githubusercontent.com/848085/39572182-2173fe5e-4ec6-11e8-9088-7f11d246426b.gif)

**Comment by LZoog** [comment]
Patch: #2911
